yapapi/payload/vm.py

```diff
@@ -72,20 +74,25 @@ async def decorate_demand(self, demand: DemandBuilder):
 
 
 async def repo(
-    *, image_hash: str, min_mem_gib: float = 0.5, min_storage_gib: float = 2.0
+    *,
+    image_hash: str,
+    min_mem_gib: float = 0.5,
+    min_storage_gib: float = 2.0,
+    min_available_logical_cpu_cores: int = 1,
```

Comment:
I think the docstring may explain the _exact_ meaning of this parameter without us having to specify such a long name for it ... 

```suggestion
    min_threads: int = 1,
```

similarly, above you have `min_mem_gib` and `min_storage_gib` and not `min_available_random_access_memory_in_gigabytes` or `min_available_storage_space` ...

Reply:
`min_threads` is very ambiguous, IMO `min_logical_cpus` would be better.

Every word in `min_available_logical_cpu_cores` (unlike in your example with `random_access_memory` instead of `ram`) is important:

- `available`: often only 1 logical cpu is shared with the requestor on a 8-core machine with 16 logical cores (a maximum value is in this case 16 - 1 = 15 available CPU cores).
- `logical_cpu_core`: it's better than `thread`, which suggests an OS thread.

Maybe: `min_offered_logical_cores`? It's ~20% shorter than the original version.

Reply:
why not `min_logical_cpus` ?

Reply:
@filipgolem @shadeofblue parameter name `min_cpu_threads` would be consistent with the name of the corresponding property (as defined in https://github.com/golemfactory/golem-architecture/blob/9e3c87790d93afff7932d6256d933faf9f9bd6d0/standards/0-commons/golem/inf/cpu.md).

For the property `golem.inf.mem.gib` we have `min_mem_gib` in `vm.repo`, for `golem.inf.storage.gib` we have `min_storage_gib` -- so for `golem.inf.cpu.threads` we could have `min_cpu_threads`.

Reply:
I agree with @azawlocki ...

Reply:
Changed to `min_cpu_threads`.